views/bigquery.go

```diff
@@ -80,8 +80,8 @@ type ViewershipEventRow struct {
 
 	// metric data
 
-	ViewCount        int64                `bigquery:"view_count"`
-	PlaytimeMins     float64              `bigquery:"playtime_mins"`
+	ViewCount        bigquery.NullInt64   `bigquery:"view_count"`
+	PlaytimeMins     bigquery.NullFloat64 `bigquery:"playtime_mins"`
```

Comment:
Any idea how these can ever be null?

Not that I think we shouldn't make this fix, but these are the most basic fields that should always be there, defaulting to `0`, so it kinda looks like a data pipeline problem.